src/script/lua_api/l_object.cpp

```diff
@@ -528,7 +518,7 @@ int ObjectRef::l_set_animation_frame_speed(lua_State *L)
 	if (sao == nullptr)
 		return 0;
 
-	if (!lua_isnil(L, 2)) {
+	if (!lua_isnoneornil(L, 2)) {
```

Comment:
API documentation needs to be adjusted accordingly to state that the argument is now entirely optional, not just "nilable"
-> `set_animation_frame_speed([frame_speed])`